app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    commands = command.split()
+    if commands[0] == "cp" and commands[1] != commands[2]:
+        with (
+            open(commands[1], "r") as file_in,
+            open(commands[2], "w") as file_out
```

Comment:
no need to use indexes here, use split to unpack vlues from splitted string to variables, like:
`var1, var2, var3 = ur_string.split()`